lib/shopify_api/resources/fulfillment.rb

```diff
@@ -6,8 +6,32 @@ def order_id
       @prefix_options[:order_id]
     end
 
-    def cancel; load_attributes_from_response(post(:cancel, {}, only_id)); end
+    def cancel(using_fulfillment_orders: false)
+      if using_fulfillment_orders
+        fulfillmentV2 = FulfillmentV2.new(attributes)
+        result = fulfillmentV2.cancel
+        load(fulfillmentV2.attributes, false, true)
+        result
+      else
+        load_attributes_from_response(post(:cancel, {}, only_id))
+      end
+    end
+
     def complete; load_attributes_from_response(post(:complete, {}, only_id)); end
     def open; load_attributes_from_response(post(:open, {}, only_id)); end
+
+    def create
```

Comment:
I'm unsure what's provided by `ActiveResource::Base` but is this intended to override `self.create` the class method instead of the instance method? What about `.save`? We still need to support both `POST /admin/api/2020-01/orders/#{order_id}/fulfillments.json` and `POST /admin/api/2020-01/fulfillments.json` no?

Reply:
Yup, that makes sense, I forgot about `save` being used for create.